card/mod/tinymce_editor/lib/javascript/script_tinymce_config.js.coffee

```diff
@@ -19,17 +19,20 @@ $.extend decko,
     # verify_html: false -- note: this option needed for empty
     #                             paragraphs to add space.
     conf = {
+      theme: "modern",
       plugins: 'autoresize'
-      autoresize_max_height: 500
+      autoresize_max_height: 500,
     }
     user_conf = if decko.tinyMCEConfig? then decko.tinyMCEConfig else {}
     hard_conf = {
-      mode: 'exact'
-      elements: el_id
+      selector: "##{el_id}",
+      branding: false
       # CSS could be made optional, but it may involve migrating old legacy
       # *tinyMCE settings to get rid of stale stuff.
       content_css: decko.cssPath
       entity_encoding: 'raw'
     }
     $.extend conf, user_conf, hard_conf
+    tinyMCE.baseURL = '/assets/tinymce'
```

Comment:
so we're still doing this, eh?  any plans to move away from this approach?

Reply:
no idea how

Reply:
yeah, me neither.